02-starting-with-data.Rmd

```diff
@@ -481,7 +481,7 @@ surveys$plot_type <- factor(surveys$plot_type)
 > 2. Can you predict the class for each of the columns in the following example?
 >    Check your guesses using `str(country_climate)`:
 >      * Are they what you expected?  Why? Why not?
->      * What would have been different if we had added `stringsAsFactors = FALSE` to this call?
+>      * What would have been different if we had added `stringsAsFactors = FALSE` to the str() function?
```

Comment:
I think the point of this question is to ask what would have been different if `stringsAsFactors = FALSE` was added to the `read.csv()` call, not the `str()` call. I don't think `stringsAsFactors` is a valid input for `str()`.

Reply:
My bad! The call is question is data.frame() so changing the text so it refers to "data.frame()" as a function may be extra confusing given the lesson content.  Maybe leave as is or simply replace "call" with "function" without specifying the function name.?